CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 6.12.3 (2022-03-01)
```

Comment:
Need to update the year!
```suggestion
## 6.12.3 (2023-03-01)
```

Reply:
Thanks!

Reply:
Addressed in https://github.com/jazzband/pip-tools/pull/1817